contracts/HubPool.sol

```diff
@@ -26,13 +31,40 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool isEnabled;
     }
 
-    WETH9Like public l1Weth;
+    enum RefundRequestStatus {
+        Pending,
+        Finalized,
+        Disputed
+    }
+
+    struct RelayerRefundRequest {
+        bytes32 poolRebalanceProof;
+        bytes32 destinationDistributionProof;
+        RefundRequestStatus status;
+    }
+
+    RelayerRefundRequest[] public relayerRefundRequests;
 
     // Whitelist of origin token to destination token routings to be used by off-chain agents.
     mapping(address => mapping(uint256 => address)) public whitelistedRoutes;
 
     mapping(address => LPToken) public lpTokens; // Mapping of L1TokenAddress to the associated LPToken.
 
+    // Address of L1Weth. Enable LPs to deposit/receive ETH, if they choose, when adding/removing liquidity.
+    WETH9Like public l1Weth;
+
+    // Address of the bridgeAdmin that is used to manage the hub.
+    BridgeAdminInterface public bridgeAdmin;
+
+    // Token used to bond the data worker for proposing relayer refund bundles.
+    IERC20 public bondToken;
+
+    // The bondToken's final fee from the UMA Store is scaled by this number to increase the bonding amount.
+    uint256 public bondTokenFinalFeeMultiplier;
```

Comment:
Can we make this a uint64?

Reply:
for context, I use uint64 for all percentage values in spokepool

Reply:
but ye we can. I'll change the type to uint64